youtube_dl/downloader/external.py

```diff
@@ -192,15 +195,19 @@ def supports(cls, info_dict):
     def available(cls):
         return FFmpegPostProcessor().available
 
+    def get_execname(self):
+        return self.execname
```

Comment:
1. It breaks if called before `_call_downloader`.
2. It not store state.
3. It should be implemented similar to `available`.